### ISSUE TITLE
Column layout breaking landscape mode

### ISSUE THREAD

**Issue by QuackingPlums** [markgdev/home-assistant_OctopusAgile#53]
On the mobile view (both in the browser and in the app) the columns break out of the card when in landscape mode.

**To Reproduce**
Add the rate card to dashboard
Rotate phone sideways

**Expected behaviour**
Not an easy fix from usability point of view, but as a minimum, elements should not break outside of the card boundary. Reducing the size of the elements further may well give us an unreadable card, and forcing single-column layout will likely make the card too long. Does HA allow scrollable cards? What would be the preferred user experience here?

**Screenshots**
![5C3497CA-726F-4D0E-AD87-2AF3F2623C5D_1_201_a](https://user-images.githubusercontent.com/3367036/105020064-bcfb7380-5a3e-11eb-8a96-4951fee6a613.jpeg)

**Smartphone (please complete the following information):**
 - iPhone 12 Pro Max
 - iOS 14.2.1
 - Safari, HA app

**Additional context**
HA 2021.1.3

**Comment by richardshaylor** [comment]
Out of interest do you get the same behaviour when using a different browser on your iPhone?

**Comment by QuackingPlums** [comment]
Yep, same happens in Chrome on iOS. It seems to be when Lovelace decides that the screen is wide enough to have more than one column, but this card disagrees.
